frontend/src/components/timeline/stylesheet.styl

```diff
@@ -24,6 +24,10 @@ $right-width = 360px
   text-align: center
   overflow: hidden
 
+  
+  & > * > * > * > * > * > * > * > div
```

Comment:
@JoelAtDeluxe if you know something I don't, please let me know! Here's what I've tried.

The dom structure of xterm is fairly different from before, which meant that we were picking up an unwanted `text-align: center`.

This feels more open ended than I would like, in that this selector hits the `code` snippet as well, but that's already left aligned. 

This div specifically doesn't have a class name, but I tried to target the div above it which has a classname of `xterm-rows` (using `& > * > * > * > * > * > * > div.xterm-rows`); however, I couldn't get it to work. I thought it might be how [styl](https://github.com/tj/styl) handles selectors, but I got similar code to work in other components; I think it must have to do with how the styling is handled with npm packages or something. 

I also tried styling the class directly, but their stylesheets overrode ours, so it didn't change anything

Reply:
Yeah, xterm is tricky. Some of the attributes of the terminal are controlled through the code rather than the styling. That makes our dynamic views a bit harder to implement. I haven't had a chance to take a look at what changed, but I can try to do so tonight.

Reply:
yeah I did notice that the a lot of css in xterm is programmatically generated, which is probably causing the issue of specifically targeting the div

Reply:
FWIW this change looks visually correct in my dev instance

Reply:
@jkennedyvz thanks for verifying! It definitely works, though it's less than ideal. I feel fine merging it if you guys do